CodeEdit/Documents/WorkspaceDocument.swift

```diff
@@ -157,6 +211,18 @@ final class WorkspaceDocument: NSDocument, ObservableObject, NSToolbarDelegate {
         selectionState.openedExtensions.remove(at: idx)
     }
 
+    /// Makes the temporary tab permanent when a file save or edit happens.
+    @objc func editorUpdatedContent() {
+        if selectionState.selectedId == selectionState.temporaryTab
+            && selectionState.temporaryTab != nil {
+            let tabID: TabBarItemID = selectionState.temporaryTab!
```

Comment:
Please use `guard` instead of force unwrap.

Reply:
Got it!